src/ripple/app/ledger/LedgerMaster.h

```diff
@@ -256,7 +256,7 @@ class LedgerMaster
     }
 
 private:
-    using ScopedUnlockType = GenericScopedUnlock <std::recursive_mutex>;
+    using ScopedUnlockType = ScopedUnlock<std::recursive_mutex>;
```

Comment:
With class template argument deduction in C++17 you no longer need this declaration.  I suggest you remove it and use class template argument deduction to construct `ScopedUnlock` objects directly.

Assuming you do that, you should also remove the `#include ... ScopedLock.h>` from this file. Then you'll need to add the include into LedgerMaster.cpp.